### DISCUSSION TITLE
tianliGPT

### DISCUSSION THREAD

**Discussion by forsakeby** [Qexo/Qexo#372] [General]
如何使用tianliGPT添加文章摘要呢，更新完弹出的框是要输入api的吗，我跳过了，还能在哪里输入呢？

**Comment by am-abudu** [comment]
设置->文章截取配置

**Reply by forsakeby** [reply]
请问会支持直接调用openai的api吗，tianliGPT绑定了自己的域名，没法生成了

**Reply by am-abudu** [reply]
难度很大。
如果是直接在浏览器端请求的话，受限于国内的网络难以联通。
如果是服务端请求的话，受限于Vercel的限制必然超时。
必须要借助第三方服务。

**Reply by forsakeby** [reply]
明白了，非常感谢
